compiler/sc1.cpp

```diff
@@ -5391,6 +5391,13 @@ static void statement(int *lastindent,int allow_decl)
   case tDECL:
   case tSTATIC:
   case tNEW:
+    if (matchtoken(tSYMBOL)) {

```

Comment:
I think you want this to be `if (tok == tNEW && matchtoken(tSYMBOL)) {` otherwise something like `static MyMethodMap()` would go into doexpr.